core/libp2p/crypto/proto/key_type.proto

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+syntax = "proto3";
+package libp2p.crypto.proto;
+
+enum KeyType {
```

Comment:
I think now it is better to have a single proto file for everything needed for crypto

Reply:
Done. Created keys.proto containing enum and keys.